transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/LibraryTransformations.scala

```diff
@@ -36,4 +37,17 @@ object LibraryTransformations {
       biosampleId = CommonTransformations.transformId(libraryInput.read[String]("biosample"))
     )
   }
+
+  /** Transform a raw ENCODE library into our preferred library-prep schema. */
+  def transformLibraryPrep(libraryInput: Msg): LibraryPrepActivity = {
+    val id = CommonTransformations.readId(libraryInput)
+    LibraryPrepActivity(
+      id = id,
+      libraryMaterial = libraryInput.tryRead[String]("nucleic_acid_term_id"),
+      libraryMaterialName = libraryInput.tryRead[String]("nucleic_acid_term_name"),
+      libraryId = id,
```

Comment:
Having two columns with the same value feels weird, but I don't know how we would combine them and/or remove one, without just combining the Library Prep table with the Library table. 

Are we keeping them separate because they will be updated in different ways? I hadn't realized that there was a 1-1 relationship between them, so I want to make sure I understand the reasoning behind separating them.

Reply:
I have similar confusion. I think that having separate tables is the most literal translation of the data model, but it might not make sense for ENCODE.